vendor/manifest

```diff
@@ -18,6 +18,15 @@
 			"notests": true
 		},
 		{
+			"importpath": "github.com/maruel/panicparse/stack",
+			"repository": "https://github.com/maruel/panicparse",
```

Comment:
Do we need it, technically no. There is always a trade off. We have two options.

1. Use a library that is specifically designed for this purpose, is already widely used, and comes with tests.  We can be fairly confident in it being correct, and to receive bug fixes and updates when things change. It does however add a new dependency, which on my laptop results in a binary that is ~100kb larger.
2. We could copy some of the library and tests to this repo. We get a smaller binary for considerably more work, both in the short term and long term.

I generally prefer option 1.